packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -755,88 +720,39 @@ abstract class BaseFlutterTask extends DefaultTask {
 
         intermediateDir.mkdirs()
 
-        if (buildMode == "profile" || buildMode == "release") {
-            project.exec {
-                executable flutterExecutable.absolutePath
-                workingDir sourceDir
-                if (localEngine != null) {
-                    args "--local-engine", localEngine
-                    args "--local-engine-src-path", localEngineSrcPath
-                }
-                args "build", "aot"
-                args "--suppress-analytics"
-                args "--quiet"
-                args "--target", targetPath
-                args "--output-dir", "${intermediateDir}"
-                args "--target-platform", "${targetPlatform}"
-                if (extraFrontEndOptions != null) {
-                    args "--extra-front-end-options", "${extraFrontEndOptions}"
-                }
-                if (extraGenSnapshotOptions != null) {
-                    args "--extra-gen-snapshot-options", "${extraGenSnapshotOptions}"
-                }
-                args "--${buildMode}"
-            }
-        }
-
         project.exec {
             executable flutterExecutable.absolutePath
             workingDir sourceDir
-
             if (localEngine != null) {
                 args "--local-engine", localEngine
                 args "--local-engine-src-path", localEngineSrcPath
             }
-            args "build", "bundle"
-            args "--target", targetPath
-            args "--target-platform", "${targetPlatform}"
+            args "--suppress-analytics"
             if (verbose) {
                 args "--verbose"
+            } else {
+                args "--quiet"
             }
-            if (fileSystemRoots != null) {
-                for (root in fileSystemRoots) {
-                    args "--filesystem-root", root
-                }
-            }
-            if (fileSystemScheme != null) {
-                args "--filesystem-scheme", fileSystemScheme
-            }
-            if (trackWidgetCreation) {
-                args "--track-widget-creation"
-            }
-            if (compilationTraceFilePath != null) {
-                args "--compilation-trace-file", compilationTraceFilePath
-            }
-            if (createPatch) {
-                args "--patch"
-                args "--build-number", project.android.defaultConfig.versionCode
-                if (buildNumber != null) {
-                    assert buildNumber == project.android.defaultConfig.versionCode
-                }
-            }
-            if (baselineDir != null) {
-                args "--baseline-dir", baselineDir
-            }
+            args "assemble"
```

Comment:
We track add2app calls via the `flutter build bundle` call, how are we tracking that now?

Reply:
build bundle is/was called by all APK building, how are you using that to track add2app?

Reply:
We track a direct call to Gradle, which then calls `build bundle` as a signal for add2app.  https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/module_test.dart#L218

Reply:
That seems incorrect - I could build a regular APK while calling gradle directly and that would show up as an add2app call?

Reply:
Updated assemble to provide the same information that build bundle did